common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2752,6 +2752,11 @@
         "message" : [
           "A type of keys and values in `map()` must be string, but got <mapType>."
         ]
+      },
+      "NULL_VALUE" : {
```

Comment:
How about to re-use the existing error condition: `NULL_DATA_SOURCE_OPTION`?

Reply:
i actually missed this condition, it makes more sense